dub.json

```diff
@@ -1,21 +1,40 @@
 {
     "name": "dateparser",
-    "description": "Library for parsing randomly formated date strings",
-    "sourcePaths": ["src/"],
+    "description": "Library for parsing randomly formatted date strings",
     "license": "BSL-1.0",
     "authors": ["Jack Stouffer"],
     "dependencies": {
         "emsi_containers": "0.4.3"
     },
     "configurations": [
         {
+            "name": "library",
+            "targetType": "library"
+        },
+        {
             "name": "unittest",
             "mainSourceFile": "bench.d",
             "targetType": "executable"
         },
         {
-            "name": "library",
-            "targetType": "library"
+            "name": "benchmark",
+            "mainSourceFile": "bench.d",
+            "targetType": "executable",
+            "dflags-ldc": ["-O5", "-singleobj", "-boundscheck=off"]
```

Comment:
could also drop that entirely, as `dub` defaults to using `-O` which is equivalent to `-O3`